Tests/Unit/Controller/Admin/ModuleSupportTest.php

```diff
@@ -0,0 +1,74 @@
+<?php
+/**
+ * Shop System Plugins:
+ * - Terms of Use can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/_TERMS_OF_USE
+ * - License can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/LICENSE
+ */
+
+use Wirecard\Oxid\Controller\Admin\ModuleSupport;
+use Wirecard\Oxid\Core\Helper;
+
+class ModuleSupportTest extends \Wirecard\Test\WdUnitTestCase
+{
+    /**
+     * @var ModuleSupport
+     */
+    private $_moduleSupport;
+
+    protected function setUp()
+    {
+        $this->_moduleSupport = oxNew(ModuleSupport::class);
+        parent::setUp();
+    }
+
+    protected function failOnLoggedExceptions()
```

Comment:
we can not/should not do that, it is class and test specific. OXID test failure handling relies on log outputs and we should only "deactivate" it where needed. Specifically here, we need to do it because we log an error then we cannot send the email and OXID treats that as a failure for the test.

Reply:
we also cannot use the oxid helper function `assertLoggedException`. It fails on PHP7.2 because they call `assertContains` with invalid arguments.

Reply:
ok